### PR TITLE
Update ios-configure-features.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the Note

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11338

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/ios-configure-features.md [hunk 1]
```diff
@@ -301,6 +301,7 @@ Defender for Endpoint on iOS enables admins to configure custom indicators on iO
 
 > [!NOTE]
 > Defender for Endpoint on iOS supports creating custom indicators only for IP addresses and URLs/domains.
+> For iOS, no alerts are generated on Microsoft 365 Defender when the URL or IP set in the indicator is accessed.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/ios-configure-features.md:305]:
```suggestion
>
> For iOS, no alerts are generated on Microsoft 365 Defender when the URL or IP set in the indicator is accessed.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**